drivers/usb/gadget/Kconfig

```diff
@@ -485,12 +485,12 @@ config USB_CONFIGFS_F_TCM
 	  Both protocols can work on USB2.0 and USB3.0.
 	  UAS utilizes the USB 3.0 feature called streams support.
 
-config USB_CONFIGFS_F_XAPTUM_PSOCK
-        bool "Xaptum psock driver"
+config USB_CONFIGFS_F_SCM
+        bool "Socket Control Model Driver"
 	depends on USB_CONFIGFS
-	select USB_F_XAPTUM_PSOCK
+	select USB_F_SCM
         help
-          This driver adds the psock type to the Linux kernel for use
+          This driver adds the SCM function to the Linux kernel for use
```

Comment:
This comment is still too generic (in describing the functionality) and too specific (in mentioning Xaptum).

Use the one I suggested in my prior review, or a slightly shortened version of. (I saw you did add that as a comment in the source code.  Nice!).

A shorted version:
```
Adds the Socket Control Module protocol for a USB device to manage and use Berkeley/POSIX-style sockets on the host.
```